### PR TITLE
Enhacement: Event loop init short form

### PR DESCRIPTION
**Pull request by mml5bg**
# Description

Event loop init has been replaced with short form.

## Azure DevOps PBI/Task reference

[AB#11846](https://softwaredefinedvehicle.visualstudio.com/SoftwareDefinedVehicle/_boards/board/t/DEV/Backlog%20items/?workitem=11846)
https://github.com/eclipse-velocitas/vehicle-app-python-sdk/issues/39

## Checklist

* [x] Vehicle App can be started with dapr run and is connecting to vehicle data broker
* [x] Vehicle App can process MQTT messages and call the seat service
* [ ] Vehicle App can be deployed to local K3D and is running
* [ ] Created/updated tests, if necessary. Code Coverage percentage on new code shall be >= 70%.
* [ ] Extended the documentation in Velocitas repo
* [ ] Extended the documentation in README.md

* [ ] Devcontainer can be opened successfully
* [ ] Devcontainer can be opened successfully behind a corporate proxy
* [ ] Devcontainer can be re-built successfully

* [ ] Release workflow is passing

### CODE REVIEW

File: app/src/main.py [hunk 1]
```diff
@@ -122,7 +122,4 @@ async def main():
     await vehicle_app.run()
 
 
-LOOP = asyncio.get_event_loop()
-LOOP.add_signal_handler(signal.SIGTERM, LOOP.stop)
```
Comment (dennismeister93) [inline app/src/main.py:126]: You can remove the unused import on line 20

Comment (mml5bg) [reply app/src/main.py:126]: Done!

**Review by dennismeister93 — APPROVED** [review]
lgtm!

File: app/src/main.py [hunk 2]
```diff
@@ -122,7 +121,4 @@ async def main():
...
-LOOP.run_until_complete(main())
-LOOP.close()
+asyncio.run(main())
```
Comment (dennismeister93) [inline app/src/main.py:124]: And another one i just found. please add a new line to the end :)

Comment (mml5bg) [reply app/src/main.py:124]: There is new line on the end, but somehow is not visible here.

Comment (dennismeister93) [reply app/src/main.py:124]: ok than i didnt say anything :)

**Pull request merged**